### PR TITLE
[ECS] Unify `tags` workflow in `instance_v2`

### PR DESCRIPTION
**Pull request by lego963**
## Summary of the Pull Request
Unify usage of tags in `resource/opentelekomcloud_compute_instance_v2`

## PR Checklist

* [x] Refers to: #918
* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
=== RUN   TestAccComputeV2Instance_basic
--- PASS: TestAccComputeV2Instance_basic (127.26s)
=== RUN   TestAccComputeV2Instance_multiSecgroup
--- PASS: TestAccComputeV2Instance_multiSecgroup (109.76s)
=== RUN   TestAccComputeV2Instance_bootFromImage
--- PASS: TestAccComputeV2Instance_bootFromImage (57.53s)
=== RUN   TestAccComputeV2Instance_bootFromVolume
--- PASS: TestAccComputeV2Instance_bootFromVolume (56.39s)
=== RUN   TestAccComputeV2Instance_changeFixedIP
--- PASS: TestAccComputeV2Instance_changeFixedIP (58.67s)
=== RUN   TestAccComputeV2Instance_bootFromVolumeVolume
--- PASS: TestAccComputeV2Instance_bootFromVolumeVolume (83.05s)
=== RUN   TestAccComputeV2Instance_stopBeforeDestroy
--- PASS: TestAccComputeV2Instance_stopBeforeDestroy (68.40s)
=== RUN   TestAccComputeV2Instance_metadata
--- PASS: TestAccComputeV2Instance_metadata (96.51s)
=== RUN   TestAccComputeV2Instance_timeout
--- PASS: TestAccComputeV2Instance_timeout (61.58s)
=== RUN   TestAccComputeV2Instance_autoRecovery
--- PASS: TestAccComputeV2Instance_autoRecovery (91.52s)
=== RUN   TestAccComputeV2Instance_crazyNICs
--- PASS: TestAccComputeV2Instance_crazyNICs (208.25s)
PASS

Process finished with exit code 0
```

### CODE REVIEW

File: opentelekomcloud/acceptance/ecs/resource_opentelekomcloud_compute_instance_v2_test.go [hunk 1]
```diff
@@ -193,72 +113,17 @@ func TestAccComputeV2Instance_bootFromVolumeVolume(t *testing.T) {
...
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckComputeV2InstanceExists(
-						"opentelekomcloud_compute_instance_v2.instance_1", &newInstance),
-					testAccCheckComputeV2InstanceInstanceIDsDoNotMatch(&instance, &newInstance),
-				),
-			},
-		},
-	})
-}
-
```
Comment (outcatcher) [inline opentelekomcloud/acceptance/ecs/resource_opentelekomcloud_compute_instance_v2_test.go]: Why those tests were deleted?

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: opentelekomcloud/acceptance/ecs/resource_opentelekomcloud_compute_instance_v2_test.go (hunk 1 above)
Comment (lego963) [reply opentelekomcloud/acceptance/ecs/resource_opentelekomcloud_compute_instance_v2_test.go]: These tests check if instance recreated. They change arguments with `ForceNew: true` set. From my pov they are redundant.

Comment (outcatcher) [reply opentelekomcloud/acceptance/ecs/resource_opentelekomcloud_compute_instance_v2_test.go]: In that case it makes sense to remove `update` step only, not the whole test.

Comment (outcatcher) [reply opentelekomcloud/acceptance/ecs/resource_opentelekomcloud_compute_instance_v2_test.go]: Or even just remove `testAccCheckComputeV2InstanceInstanceIDsDoNotMatch`

**Review by outcatcher — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**